.gitattributes

```diff
@@ -0,0 +1,11 @@
+* text=auto
```

Comment:
that would be a good opportunity to add `eol=lf`, too!

Reply:
Yes, your are right